docroot/sites/settings/cgov_core.settings.php

```diff
@@ -63,12 +63,14 @@
 switch ($env) {
   case '01live':
```

Comment:
Given this issue, can we move 01live below the dev blocks? Just that would have highlighted the missing break.